kubefed/installing-kubefed.adoc

```diff
@@ -0,0 +1,23 @@
+[id="installing-kubefed"]
+= Installing OpenShift KubeFed
+include::modules/common-attributes.adoc[]
+include::modules/kubefed-document-attributes.adoc[]
+:context: installing-kubefed
+
+To get started with {KubeFedProductName}, you must complete the Operator installation and then create the required custom resources (CRs).
+
+NOTE: {KubeFedProductName} can be deployed as either a namespace-scoped or a cluster-scoped deployment, but not both, as this will cause conflicts when both deployments attempt to manage the same set of resources.
+
+== Installing the {KubeFedProductName} Operator
+
+You can install the {KubeFedProductName} Operator in the host cluster by following the {product-title} instructions on installing an Operator.
+
+For details, see the {product-title} documentation on xref:../applications/operators/olm-adding-operators-to-cluster.adoc[adding Operators to a cluster].
```

Comment:
We should establish in this material that we expect users that want cluster-scoped kubefed to:

- Create the openshift-federation-system namespace
- Install the operator into that namespace
- Create the `KubefedWebhook` CR in that namespace
- Create the `Kubefed` CR in that namespace
- Ensure that the resource examples we ask people to create are within that namespace

...and that if you want namespace-scoped federation you should:

- Create the openshift-federation-system namespace
- Install the operator into that namespace
- Create the `KubefedWebhook` CR in that namespace
- Create the `Kubefed` CR in the namespaces that you want ns-level kubefed in.

Reply:
`oc create ns openshift-federation-system`

Reply:
For the namespace scoped federation control planes, we need to make sure that the operator instance is observing all namespaces for creation of the KubeFed CR and reconciling them?

Reply:
I prefer this part is added to the kubefed operator. I mean when we install kubefed operator from operatorhub, we can see the description about how to install kubefed.
And there is a bug for this: https://bugzilla.redhat.com/show_bug.cgi?id=1731845